tripal_chado/includes/TripalFields/sbo__relationship/sbo__relationship.inc

```diff
@@ -355,14 +427,15 @@ class sbo__relationship extends ChadoField {
     $entity->{$field_name}['und'][$delta]['type_name'] = $relationship->type_id->name;
     $entity->{$field_name}['und'][$delta]['subject_name'] = $subject_name . ' [id: ' . $relationship->$subject_id_key->$subject_pkey . ']';
     $entity->{$field_name}['und'][$delta]['object_name'] = $object_name  . ' [id: ' . $relationship->$object_id_key->$object_pkey . ']';
-    if (array_key_exists('value', $schema['fields'])) {
+    if (array_key_exists('value', $this->schema['fields'])) {
       $entity->{$field_name}['und'][$delta]['chado-' . $field_table . '__value'] = $relationship->value;
     }
-    if (array_key_exists('rank', $schema['fields'])) {
+    if (array_key_exists('rank', $this->schema['fields'])) {
       $entity->{$field_name}['und'][$delta]['chado-' . $field_table . '__rank'] = $relationship->rank;
     }
   }
- /**
+
+  /**
    *
    * @see TripalField::load()
    */
```

Comment:
below, `     $bundle = tripal_load_bundle_entity(array('name' => $entity->bundle));` is called but `$bundle` is not used.  Let's not load the bundle if we don't use it, i think that call isn't cheap

Reply:
Good catch! Removed 👍